src/wasm-type-shape.h

```diff
@@ -173,6 +174,10 @@ struct UniqueRecGroups {
   // Otherwise rebuild the group  make it unique and return the rebuilt types,
```

Comment:
```suggestion
  // Otherwise rebuild the group to make it unique and return the rebuilt types,
```
(pre-existing, but just noticed this now)